src/main/java/com/knu/buga1chuk/interfaces_practice/Demo.java

```diff
@@ -0,0 +1,28 @@
+package com.knu.buga1chuk.interfaces_practice;
+
+import com.knu.buga1chuk.interfaces_practice.model.Alien;
+import com.knu.buga1chuk.interfaces_practice.model.Animal;
+import com.knu.buga1chuk.interfaces_practice.model.Person;
+import com.knu.buga1chuk.interfaces_practice.service.AlienInfoService;
+import com.knu.buga1chuk.interfaces_practice.service.AnimalInfoService;
+import com.knu.buga1chuk.interfaces_practice.service.InfoService;
+import com.knu.buga1chuk.interfaces_practice.service.PersonInfoService;
+
+public class Demo {
+    public static void main(String[] args) {
+        Animal animal = new Animal(1);
+        Person person = new Person("Bob");
+        Alien alien = new Alien(51);
+
+        InfoService<Alien> alineInfoService = new AlienInfoService();
+        InfoService<Animal> animalInfoService = new AnimalInfoService();
+        InfoService<Person> personInfoService = new PersonInfoService();
+
+        alineInfoService.showInfo(alien);
+        animalInfoService.showInfo(animal);
+        personInfoService.showInfo(person);
```

Comment:
Check the `greetings` and `sleep` methods as well

Reply:
fix